internal/topo/node/source_node.go

```diff
@@ -111,6 +111,7 @@ func (m *SourceNode) Open(ctx api.StreamContext, errCh chan<- error) {
 			props["delimiter"] = m.options.DELIMITER
 			m.options.Schema = nil
 			if m.schema != nil {
+				m.options.SchemaKey = ctx.GetRuleId()
```

Comment:
The name of schemaKey is misleading. Since the rule id is already stored in context, why do we need to re-define it here?